democracylab/settings.py

```diff
@@ -215,7 +214,7 @@ def read_connection_config(config):
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
 PROTOCOL_DOMAIN = os.environ['PROTOCOL_DOMAIN']
-ADMIN_EMAIL = os.environ['ADMIN_EMAIL']
+ADMIN_EMAIL = os.getenv('ADMIN_EMAIL')
```

Comment:
Any reason for this change?  ADMIN_EMAIL is a required environment variable, so it's okay if a null variable throws an exception